packages/gatsby/src/utils/websocket-manager.js

```diff
@@ -148,20 +151,31 @@ class WebsocketManager {
         }
       }
 
-      s.on(`registerPath`, path => {
-        s.join(getRoomNameFromPath(path))
-        activePath = path
-        this.activePaths.add(path)
-
+      const getDataForPath = path => {
         if (!this.pageResults.has(path)) {
           const result = getCachedPageData(path, this.programDir)
-          this.pageResults.set(path, result)
+          if (result) {
+            this.pageResults.set(path, result)
+          } else {
+            console.log(`Page not found`, path)
```

Comment:
I'm seeing this error output even though the page is loading.

Clicking around the blog, and then the tags page (http://localhost:8000/blog/tags) and individual tag pages

```
View GraphiQL, an in-browser IDE, to explore your site's data and schema

  http://localhost:8000/___graphql

Note that the development build is not optimized.
To create a production build, use gatsby build

boundActionCreators is deprecated. Use actions instead. Check the following files:

src/pages/docs/actions.js

ℹ ｢wdm｣:
ℹ ｢wdm｣: Compiled successfully.
Page not found /blog/tags/documentation
Page not found /blog/tags/launch
Page not found /blog/tags
Page not found /blog/tags
Page not found /blog/tags
Page not found /blog/tags
```

Reply:
This is fixed by https://github.com/gatsbyjs/gatsby/pull/6777/commits/5fdcc4ca68fe4b50e596d2c480f01b751cdba3b1